torchgeo/models/resnet.py

```diff
@@ -37,6 +38,9 @@
     K.CenterCrop(224),
     K.Normalize(mean=_min, std=_max - _min),
     K.Normalize(mean=torch.tensor(0), std=1 / torch.tensor(255)),
+    Lambda(lambda x: torch.clamp(x, min=0.0, max=255.0)),
+    Lambda(lambda x: x.to(torch.uint8).to(torch.float)),  # type: ignore[no-any-return]
```

Comment:
Does this actually do anything?

Reply:
Yes it converts from float to uint8 to reduce bit resolution. The conversion back to float is so that the other kornia augmentations don't complain about it being float

Reply:
This feels like the same situation as the clip above. It reduces information from the image and makes the job harder for downstream tasks. This may help during SSL because image comparison is harder, but I have no a priori reason to believe that this would help with downstream tasks. If you perform an ablation study and find that it helps, I'm fine with adding it.

Reply:
I agree with you that it's bad to throw away information, but we should make another transform function if we want to remove certain pieces. Our current seco transform isn't correct which is what we are advertising by attaching it to the weights.